webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -137,20 +140,29 @@ const ActionsBar = ({
 
   return (
     <>
-      <Button
-        onClick={() => visit(foremanUrl(`/hosts/${hostId}/edit`))}
-        variant="secondary"
-        isDisabled={!canEdit}
-      >
-        {__('Edit')}
-      </Button>
-      <Dropdown
-        alignments={{ default: 'right' }}
-        toggle={<KebabToggle id="hostdetails-kebab" onToggle={onKebabToggle} />}
-        isOpen={kebabIsOpen}
-        isPlain
-        dropdownItems={dropdownItems.concat(registeredItems)}
-      />
+      <Split hasGutter>
+        <SplitItem>
+          <Slot hostId={hostId} id="_rex-host-features" />
+        </SplitItem>
```

Comment:
Cant that be a slot for more then one action? Like follows:
```suggestion
        <Slot hostId={hostId} id="host-toolbar-actions" />
```